scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/headers/PipelineHeaderSetter.kt

```diff
@@ -0,0 +1,21 @@
+package io.seldon.dataflow.kafka.headers
+
+import io.seldon.dataflow.kafka.TRecord
+import org.apache.kafka.streams.kstream.ValueTransformer
+import org.apache.kafka.streams.processor.ProcessorContext
+
+class PipelineHeaderSetter(private val pipelineName: String) : ValueTransformer<TRecord, TRecord> {
+    var context: ProcessorContext? = null
+
+    override fun init(context: ProcessorContext?) {
+        this.context = context
+    }
+
+    override fun transform(value: TRecord?): TRecord? {
```

Comment:
perhaps add comment in the code to explain which header value we are replacing for `SeldonHeaders.pipelineName`